<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>staffSearchAnimals</class>
 <widget class="QMainWindow" name="staffSearchAnimals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Animals</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>781</width>
      <height>576</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QPushButton" name="HomePushButton">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>47</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>47</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="2">
          <widget class="QLabel" name="Max">
           <property name="minimumSize">
            <size>
             <width>93</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="Min">
           <property name="minimumSize">
            <size>
             <width>94</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Age">
           <property name="minimumSize">
            <size>
             <width>93</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>29</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Age</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="MaxSpinBox">
           <property name="minimumSize">
            <size>
             <width>93</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>29</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>8</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="MinSpinBox">
           <property name="minimumSize">
            <size>
             <width>94</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>29</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="maximum">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="Type">
           <property name="minimumSize">
            <size>
             <width>144</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>144</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="TypeComboBox">
           <property name="minimumSize">
            <size>
             <width>143</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>143</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fish</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Amphibian</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mammal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bird</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="Exhibit">
           <property name="minimumSize">
            <size>
             <width>144</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>144</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Exhibit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="ExhibitComboBox">
           <property name="minimumSize">
            <size>
             <width>143</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>143</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pacific</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jungle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sahara</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mountainous</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Birds</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="AtlantaZoo">
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Atlanta Zoo </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Name">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>74</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="NameLineEdit">
         <property name="minimumSize">
          <size>
           <width>249</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>249</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Species">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>74</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Species</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="SpeciesLineEdit">
         <property name="minimumSize">
          <size>
           <width>249</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>249</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="SearchPushButton">
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>93</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QTableWidget" name="AnimalTable">
       <property name="minimumSize">
        <size>
         <width>761</width>
         <height>281</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>761</width>
         <height>281</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Species</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Exhibit</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Age</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
